sos/report/plugins/qdevice.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2024 Javier Blanco <javier@jblanco.es>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+
+class Qdevice(Plugin, IndependentPlugin):
+    short_desc = 'qdevice information'
+
+    plugin_name = 'qdevice'
+    packages = ('corosync-qnetd')
```

Comment:
nitpick, please add a ',' at the end of the package

Reply:
ack

Reply:
Done in commit 43e5cc26cb7681eec0ba8af3e49696371af6bfc0